qiita_db/analysis.py

```diff
@@ -79,6 +79,26 @@ def _status_setter_checks(self, conn_handler):
             raise QiitaDBStatusError("Can't set status away from public!")
 
     @classmethod
+    def get_user_default(cls, user):
```

Comment:
The check if ID exists on Analysis instantiation isn't needed in this case. That's the DB call I'm referring to.